test/helpers/YieldOracleSharedSetup.sol

```diff
@@ -68,6 +68,7 @@ abstract contract YieldOracleSharedSetup is Test {
 
     function setUp() public virtual {
         // Warp to reasonable timestamp
+        uint256 currBlockTimeStamp = block.timestamp;
         vm.warp(1_696_181_435);
```

Comment:
You should be able to just get rid of this `vm.warp()` as long as you have it in the YieldOracle unit tests.

Reply:
The shared setup contract needs to warp on it's own in order to retrieve correct historical exchange rate values. So it warps time, retrieves the correct values, then it goes back to the original time here.